### ISSUE TITLE
AENS - Remove Avatars mechanism (seeded images) since it does not offer any value.

### ISSUE THREAD

**Issue by smaroudasunicorn** [superhero-com/superhero-wallet#3135]
This task is about removing these face-like avatar and replacing them with default geometrical ones:

![Image](https://github.com/superhero-com/superhero-wallet/assets/8217670/9f42a2ad-c868-4f46-a9ca-b6e287176098)

Also the name management page needs to be updated by totally removing the avatars placed next to the name entries:

![Image](https://github.com/superhero-com/superhero-wallet/assets/8217670/f8dcb523-f360-4b53-b058-d7deb3dea12e)

The pending indicator needs to be placed below the name:

![Image](https://github.com/superhero-com/superhero-wallet/assets/8217670/c3a18714-d64b-421f-ac68-b24078242bbd)

**Comment by Liubov-crypto** [comment]
I see that the images have been removed. 

I have a few questions about the design, I compared to attached screens and see that there are some discrepancies. Was the design strictly implemented or is it in progress? 


1. When name is pending status it will be displayed without 3 dots..
![Image](https://github.com/superhero-com/superhero-wallet/assets/69896204/c5e6c0fc-3b0f-4de0-bb9b-7d18a2a62b12)

2.  If the name already has pointer it will be not visible under, only make default / default button is available: 

https://github.com/superhero-com/superhero-wallet/assets/69896204/62effae3-3263-4bff-8bf4-4c32a47b549b

![Image](https://github.com/superhero-com/superhero-wallet/assets/69896204/e03c51fa-1109-4f11-ae3c-3a87805f736a)


3. If I add as a pointer an account address that not belong to registered name I will be not able to set this name as default (it was possible before): 



https://github.com/superhero-com/superhero-wallet/assets/69896204/5aaae557-6a70-4eed-8f95-b0cbb21e8f65

**Comment by onvisions** [comment]
@Liubov-crypto
I have synced with @peronczyk the implementation of the design. I made two minor comments on my side: 
- 3 dots should be added and it should be the same for pending state of transactions. @peronczyk told me he already fixed this.
- avatar should be removed from Auctions tab as well. 

**About the design reference:** 

There is not updated design reference for the whole AENS feature in Figma. Meaning that the existing design is outdated.
_(For example we already have the Account details screen and the AENS design is still with the tabs placed directly below the account switcher instead in the Account details modal)_

@smaroudasunicorn The reason for that is that we used to have on the roadmap Redesign of the whole AENS feature and there was no point in updating and then redesigning the whole thing. At the time PO decided we'd better focused on the redesign. However the AENS redesign was later postponed due to shift of priorities on the roadmap. 

IMO I should spend couple of days on updating Figma design reference (AENS feature) to reflect current state of the extension/app.
